node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -137,12 +137,12 @@ where
                 let response_block_number = match block_request.wait() {
```

Comment:
Could you please use the suffix `_opt` for this variable and any similarly created? As now the typing is different, and it is actually optional.

Reply:
On the other hand, I haven't come across a line where this variable could resolve as None. It seems to me it can never be None. If you agree with me, you may want to simplify it, it would also help with the arguments in `find_transactions_largest_block_number()`.  Which I'd also appreciate.